src/inputs/ChipSelectField.tsx

```diff
@@ -142,10 +142,14 @@ export function ChipSelectField<O, V extends Value>(
       if (selectedItem) {
         onSelect(key as V, selectedItem);
       }
+      // Per UX, when a field is selected then we want to call our `onBlur` callback and remove the focus styles. The field _is_ still in focus, but that is only to retain tab position in the DOM.
+      setIsFocused(false);
```

Comment:
I almost want to add a new `useState` that is like:

```
const [pretendNotFocused, setPretendNotFocused] = useState()
```

And call `setPretendNotFocused` here, and use like a `const effectivelyFocused = isFocused && !pretendNotFocused`.

Just b/c it's going to make me really confused to 6 months from now wonder wtf `isFocused` is false when the browser still thinks its focused. :-)

That said, this is probably too safe; but maybe add a comment where we declare the `isFocused` `useState` to add a disclaimer that we maybe prematurely call this.

Also I wonder if this warrants a comment on the `onBlur` prop that this is more a logical blur than the physical blur.

Reply:
...the other components seem to just call `element.blur()`; maybe a comment about why this one is different?

Reply:
Great questions around this, and great perspective. There is definitely a difference here,... we have the actual browser focus, and then our 'visual' focus state. I think separating those would be very helpful. I will work on this. Thanks!

Reply:
The `isFocused` value we are using here was really to just represent a visual focus state, so I named the variables accordingly. Please take another look when you get a chance.